### PR TITLE
blogc 0.12.0 (new formula)

### PR DESCRIPTION
**Pull request by cniemira**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/blogc.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Blogc < Formula
+  desc "Static blog compiler"
+  homepage "https://blogc.rgm.io"
+  url "https://github.com/blogc/blogc/releases/download/v0.12.0/blogc-#{version}.tar.bz2"
```
Comment (woodruffw) [inline Formula/blogc.rb]: This fails since you invoke `version` after `url`. In this case, skip `version` entirely and just put `0.12.0` in the `url`.

**Comment by cniemira** [comment]
Done! The version is now specified only in the URL, and no separate version param exists.

File: Formula/blogc.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  sha256 "421474fa089024015f7dca77386c3ca360531ae4510ae50b4dc3446653a4edbe"
+
+  def install
+    system "./configure", "--enable-runserver",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "git", "clone", "https://github.com/blogc/blogc-example.git"
```
Comment (bfontaine) [inline Formula/blogc.rb]: Use a `resource do` block instead of cloning in the test block.

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by fxcoudert** [comment]
No update in 3 months, closing for now, please reopen when the comments are addressed.

**Pull request closed without merging**